packages/react-native-test-app-msal/android/build.gradle

```
@@ -73,7 +73,11 @@ repositories {
     }
 }
 
+def rnxDisableAuthProperty = "com.microsoft.reacttestapp.msal.disableRnxAuthModule"
 def rnxAuth = findNodeModulesPath("@rnx-kit/react-native-auth")
+def skipAuthModuleRegistration =
+    rootProject.hasProperty(rnxDisableAuthProperty) &&
```

I think we should add the `rnxAuth` check here to avoid accidentally enabling auth module registration when it's not available:
```suggestion
def skipAuthModuleRegistration =
    rnxAuth != null &&
    rootProject.hasProperty(rnxDisableAuthProperty) &&
```

---

Agreed!